v21.2/restoring-backups-across-versions.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Restoring Backups Across Versions
+summary: CockroachDB's support policy for restoring backups across versions.
+toc: true
+docs_area: manage
+---
+
+This page describes the support Cockroach Labs provides for restoring backups across major versions of CockroachDB. The sections outline the support and examples for the most common use cases when restoring backups across versions:
+
+- [Support for restoring backups into a newer version](#support-for-restoring-backups-into-a-newer-version)
+- [Restore into a newer version](#restore-into-a-newer-version)
+- [Support for long-term backup archival](#support-for-long-term-backup-archival)
+
+## Support for restoring backups into a newer version
+
+Cockroach Labs supports restoring a backup on one major version following the major version on which you took the backup. Therefore, if you took a backup on major version N, you can restore it on major version N+1.
+
+For example, backups taken on v21.1 can be restored into v21.2. Backups taken on v21.2 can be restored into v22.1. The following table outlines this support:
+
+Backup taken on version | Restorable into version
+------------------------+--------------------------
+20.1                    | 20.2
+20.2                    | 21.1
+21.1                    | 21.2
+21.2                    | 22.1
+22.1                    | 22.2
+
+See the following section, [Restore into a newer version](#restore-into-a-newer-version), for an example of of restoring from multiple major versions previous to a newer version.
+
+{{site.data.alerts.callout_info}}
+Cockroach Labs does **not** support restoring backups from a newer version into a previous version. 
```

Comment:
from a higher version into a lower version? (newer is relative to date e.g. 22.1.100 may be "newer" than 22.2.0?)

Reply:
Good point... I have changed the language around this